addons/web/static/src/views/calendar/calendar_model.js

```diff
@@ -33,8 +35,9 @@ export class CalendarModel extends Model {
         const formViewIdFromConfig = formViewFromConfig ? formViewFromConfig[0] : false;
         const fieldNodes = params.popoverFieldNodes;
         const { activeFields, fields } = extractFieldsFromArchInfo({ fieldNodes }, params.fields);
+        this.state = params.state;
         this.meta = {
-            ...params,
+            ...omit(params, "state"),
```

Comment:
(related to the removal request above):
```suggestion
            ...omit(params, "state"),
            date: params.state.date || null,
```